analyses/0012-TUF/analysis.md

```diff
@@ -0,0 +1,533 @@
+  correspond with the label of the section i.e _About_ should only contain
+  introductory content about the project.
+
+- The information available for news users and contributors is not satisfactory.
+  There are no step-to-step guides and tutorials for new users to get started
+  with the software, neither are there contributor guidelines.
+
+#### Information architecture
+
+- There is an _overview_ section explaining what TUF is and its use cases. The
+  features, the metadata and roles are well explained with examples for each
+  metadata.
+
+- Repetition of content on different pages makes content confusing
+
+- Re-organise content to make it easier to follow
+
+- There are no tutorials for specific feature implementation. But, there are
+  videos explaining various use cases.
+
+- There aren't specific task-based guides for features. The available guide for
+  implementing the specification is part of a larger document labelled
+  [Specification(latest)](https://theupdateframework.github.io/specification/latest)
+
+- There is a FAQ and reporting issues sections for troubleshooting.
+
+- There is a well detailed API reference for multiple TUF APIs
+
+- README on
+  [theupdateframework.io](https://github.com/theupdateframework/theupdateframework.io)
+  repo is empty with little information about the content of the repo.
+
+#### New user content
+
+- There is a _Getting started_ section on the website with information about
+  main features. But there's repetition of content from other pages. For example
+  the _Specification (latest)_ and _index_ include information present in the
+  project and overview sections. This section is confusing for new users.
+
+- The documentation repo does not have a contributor guide for new users to get
+  started
+
+- There isin't any documentation labeled 'Installation guide'. Instead,
+  installation instructions are part of a larger document labelled
+  [The Update Framework specification](https://theupdateframework.github.io/specification/latest)
+
+- TUF docs do not provide information about application-specific/OS
+  functionality in system updates. Instead the documentation states that TUF
+  provides a secure way for applications to obtain and verify files being
+  distributed by trusted parties.
+
+- There is sample code in the content that can be easily copy-pasted on other
+  platforms.
+
+- It's not clear whether the documentation is up-to-date as there are no dates
+  on docs or release notes published.The only document on the site with dates is
+  the
+  [Specification file](https://theupdateframework.github.io/specification/latest)
+
+#### Content maintainability & site mechanics
+
+- The documentation is not searchable. You have to go through the site to find
+  what you are looking for. The only source of naviagation is the menu bar.
+
+- The Docs are managed through docs-as-code site Hugo and content written in
+  markdown. However,it appears the updates are made manually.
+
```

Comment:
@Dindihub ☝️ 

I see a bunch of other comments by @lukpueh that don't seem to have been read and/or processed by you. For each comment like this one by @lukpueh, please mark as resolved or give it a thumbs up when you are done processing it so that we'll know that you've seen it.

(I know that some comments are tagged by GH as "Outdated", but that doesn't confirm to me that you've addressed the comment -- because they can become "Outdated" by mistake.)